src/test/java/edu/kis/vh/nursery/RyhmersJUnitTest.java

```diff
@@ -8,6 +8,7 @@
 public class RyhmersJUnitTest {
 	
 	private DefaultCountingOutRyhmer ryhmer = new DefaultCountingOutRyhmer();
+	final int EMPTY_STACK_VALUE = 0;

```

Comment:
Original constant should be used (it is better to use a constant from stacks interface).